kong/router/atc_compat.lua

```diff
@@ -46,6 +46,7 @@ local get_upstream_uri_v0  = utils.get_upstream_uri_v0
 
 
 local TILDE            = byte("~")
+local ASTER            = byte("*")
```

Comment:
Let's use `ASTERISK` instead, `ASTER` is not a word.

Reply:
done